projects/fsastorefrontlib/src/shared/util/helpers/pipe/banking-entry-value.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { TranslatePipe } from '@spartacus/core';
+
+@Pipe({
+  name: 'cxFsBankingEntryValue',
+})
+export class BankingEntryValuePipe implements PipeTransform {
```

Comment:
Not sure if we really need this pipe. By default, pipes are used to transform strings, currency, dates, and other data like DatePipe, PercentPipe etc. This one is not transforming anything, it is just used to find the entry in the provided list. Can't see how this pipe can be used on different pages.
I understand that it's needed because 2 banking products can have a different list of entries and we need to find shared ones but can we handle this somehow in  quote comparison component?

Reply:
We have implemented almost the same approach with `BillingEventValuePipe`, but you are right in saying that pipes are perhaps not intended to be used in this way. However I would say, we have limited choices. We can either unpack the these objects in component model and transform them there, or we can maybe create mini components that would also have transformation in their model and then inject those components in this template. The third option is this one with the pipe. Maybe we can discuss this in more detail.